src/components/BorderCountries/BorderCountries.jsx

```diff
@@ -0,0 +1,43 @@
+import { useContext, useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import { DarkModeContext } from "../../Contexts/DarkModeContext";
+import {
+  BorderCountriesContainer,
+  BorderCountryLink,
+} from "./BorderCountries.styles";
+
+export default function BorderCountries({ borderCountries }) {
+  const { isDarkMode } = useContext(DarkModeContext);
+  const [fullNames, setFullNames] = useState([]);
+  const fetchBorderCountriesNames = async () => {
+    const response = await fetch(
+      `https://restcountries.com/v2/alpha?codes=${borderCountries}`
+    );
+    if (response.ok) {
```

Comment:
mały spoiler - jak już przejdziemy przez to code review to następny pull request poświęcimy na tworzenie hooków i plików dla api :D